### PR TITLE
feat: filter available tokens to bridge

### PR DESCRIPTION
**Pull request by 0xArdy**
closes SUP-90

### CODE REVIEW

File: src/hooks/useTokenList.tsx [hunk 1]
```diff
@@ -10,16 +10,20 @@ export const useTokenList = () => {
...
     () => tokenList.filter((token: TokenData) => token.chainId === toChain?.id),
     [toChain?.id, tokenList],
   );
 
+  const fromTokens = useMemo(() => {
+    return tokenList.filter(
+      (token: TokenData) =>
+        token.chainId === fromChain?.id &&
+        // Also the token should exist on the 'To' chain
+        toTokens.some((toToken: TokenData) => toToken.name === token.name && toToken.symbol === token.symbol),
```
Comment (turtlemoji) [inline src/hooks/useTokenList.tsx:23]: Maybe just do a regular `if` inside `{}` so it is more human readable

File: src/containers/BridgeCard/ChainSection.tsx [hunk 2]
```diff
@@ -91,9 +91,15 @@ const ChainSectionContainer = styled(Box)(() => {
...
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'center',
+      maxWidth: 'calc(100% - 0.8rem)',
+      margin: '0 auto',
+
       div: {
-        maxWidth: '50%',
-        width: '15.05rem',
+        maxwidth: 'auto',
```
Comment (turtlemoji) [inline src/containers/BridgeCard/ChainSection.tsx:101]: image
Unkown property name

File: src/containers/BridgeCard/ChainSection.tsx [hunk 3]
```diff
@@ -91,9 +91,15 @@ const ChainSectionContainer = styled(Box)(() => {
     },
 
     '@media (max-width: 600px)': {
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'center',
+      maxWidth: 'calc(100% - 0.8rem)',
```
Comment (turtlemoji) [inline src/containers/BridgeCard/ChainSection.tsx:97]: why not just `padding: 0 0.4rem;` ?

Also rn it gets weird still:
image

Comment (0xArdy) [reply src/containers/BridgeCard/ChainSection.tsx]: fixed https://github.com/defi-wonderland/superchain-bridge-ui/pull/49/commits/c1be5ca01e8d8913d56e3aae3acef5fa432e9dad

File: src/containers/BridgeCard/ChainSection.tsx (hunk 2 above)
Comment (0xArdy) [reply src/containers/BridgeCard/ChainSection.tsx]: fixed https://github.com/defi-wonderland/superchain-bridge-ui/pull/49/commits/c1be5ca01e8d8913d56e3aae3acef5fa432e9dad

File: src/hooks/useTokenList.tsx (hunk 1 above)
Comment (0xArdy) [reply src/hooks/useTokenList.tsx]: https://github.com/defi-wonderland/superchain-bridge-ui/pull/49/commits/7257c9a835a8cf61015185c528c9839733d6d8e3

**Review by turtlemoji — APPROVED** [review]

**Review by 0xtiti — COMMENTED** [review]
lgtm! should we show the dropdown option when there is only one token available?

**Comment by 0xArdy** [comment]
> lgtm! should we show the dropdown option when there is only one token available?

good point. We can talk about it in the next review!

**Review by 0xtiti — APPROVED** [review]

**Pull request merged**